packages/amplify_datastore/example/integration_test/query_test/sort_order_test.dart

```diff
@@ -104,7 +104,7 @@ void main() {
         DateTime(0000, 01, 01, 00, 00, 00),
         DateTime(1970, 01, 01, 00, 00, 00),
         DateTime(2020, 01, 01, 00, 00, 00),
-        DateTime(2020, 01, 01, 23, 59, 59),
+        DateTime(2020, 01, 02, 23, 59, 59),
```

Comment:
Was the test failing without this change? Might be a bug?

Reply:
Yes was failing, although not bug IMO. Discussed offline w @HuiSF but was failing consistently in CI on android/ios but not locally. These 2 values get converted to the same date and their order would get switched. Not sure why their order gets switched as I understand return in save in order otherwise? though not clear if that's a requirement. However, this test should just test the date order not the order when date equals.

If expected behavior is indeed that should be in some order even when the same date, there should be a separate test for that IMO, and it's a bug.

Reply:
I see. Thanks for the explanation. I thought this test was for `AWSDateTime`. I see it is for `AWSDate`. It is interesting that it was only failing in CI, but I agree there is no expectation for order in this case.